spec/functional/resource/link_spec.rb

```diff
@@ -348,14 +348,14 @@ def create_resource
               # TODO For some reason our temporary AIX 7.2 system does not correctly report user existence immediately after changes have been made.
               # Adding a 2 second delay for this platform is enough to get correct results.
```

Comment:
let's change these comments to say 'Adding a delay" so we don't have to keep comment in sync with code - the code tells us the number already.  Similarly, "AIX 7.x" should just be AIX in the comments.